CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Class hierarchy for objectives
 - Deserialization is now also possible from optional class name abbreviations
 - `Kernel` base class allowing to specify kernels
-- `MaternKernel` class can be chosen for GP surrogates
+- `MaternKernel` and `ScaleKernel` classes can be chosen for GP surrogates
```

Comment:
scale kernel shouldbe its own point, its not on the same "level" as base kernels

Reply:
Changed.